gatsby-node.ts

```diff
@@ -23,14 +24,28 @@ export const onCreateNode: GatsbyNode<SatellytesNode>['onCreateNode'] = async (
    * Provide a slug for any markdown page (which is not given by default)
    * to build our pages based on this markdown file.
    */
-  if (node.internal.type === `MarkdownRemark`) {
+  if (
+    node.internal.type === `MarkdownRemark` &&
+    Object.keys(node).includes('fileAbsolutePath')
+  ) {
     const value = createFilePath({ node, getNode });
     createNodeField({
       name: `slug`,
       node,
       value,
     });
   }
+
+  /**
+   * Provide a path for all contentful blog posts
+   */
+  if (node.internal.type === 'ContentfulBlogPost') {
+    createNodeField({
+      name: 'path',
+      node,
+      value: `/blog/${node.slug}/`,
```

Comment:
I really like that the path gets pinned to the `ContentfulBlogPost`. It feels like the `/blog` can be a constant, but I have no idea where to put this constant so it's fine to just leave it as it is 😄